crates/primitives/src/result.rs

```diff
@@ -24,7 +24,11 @@ pub enum ExecutionResult {
         output: Output,
     },
     /// Reverted by `REVERT` opcode that doesn't spend all gas.
-    Revert { gas_used: u64 },
+    Revert {
+        gas_used: u64,
+        logs: Vec<Log>,
+        return_value: Bytes,
```

Comment:
And Maybe rename it to `output` as to have similar name to `Success::output`

Reply:
Fixed in latest commit